app/controllers/replies_controller.rb

```diff
@@ -210,9 +210,18 @@ def destroy
 
     previous_reply = @reply.send(:previous_reply)
     to_page = previous_reply.try(:post_page, per_page) || 1
-    @reply.destroy # to destroy subsequent ones, do @reply.destroy_subsequent_replies
-    flash[:success] = "Post deleted."
-    redirect_to post_path(@reply.post, page: to_page)
+
+    # to destroy subsequent replies, do @reply.destroy_subsequent_replies
+    begin
+      @reply.destroy!
+      flash[:success] = "Reply deleted."
```

Comment:
Oh, neat, you fixed "Post deleted" being wrong. <3

Reply:
I was going through them and it annoyed me!